target/snitch_cluster/Makefile

```diff
@@ -190,6 +190,27 @@ $(eval $(call generate_snax_gen,snax_streamer_gemmX))
  
 endif
 
+ifeq (${CFG_OVERRIDE}, cfg/snax-streamer-gemmX-xdma.hjson)
+
+	CLUSTER_NAME = snax_streamer_gemmX_xdma_cluster
+
+$(eval $(call generate_snax_gen,snax_streamer_gemmX))
+
+	SNAX_GEMMX_ROOT ?= $(ROOT)/hw/chisel_acc
+    include $(SNAX_GEMMX_ROOT)/Makefile
+
+# This is a temporary solution. Since the xdma is now generated within wrapperergen, we need to tell Makefile about this by adding these two files to the SNAX_GEN list
```

Comment:
Why is it temporary? It seems correct though. But also needs to be inside indeed! We can make it mandatory on the top!